src/cr/cube/cube.py

```diff
@@ -431,6 +473,22 @@ def _is_single_filter_col_cube(self):
         """bool determines if it is a single column filter cube."""
         return self._cube_dict["result"].get("is_single_col_cube", False)
 
+    @lazyproperty
+    def _mean_subreferences(self):
+        """List of mean subreferences, tipically for numeric arrays."""
+        cube_response = self._cube_response
+        cube_measures = cube_response.get("result", {}).get("measures", {})
+        metadata = cube_measures.get("mean", {}).get("metadata", {})
+        return metadata.get("references", {}).get("subreferences", [])
```

Comment:
In one case we have `None` for the invalid value, while here we'll have `[]`. We need to make this uniform. Both values are fine with me, although `[]` is probably better because of the uniform type.

Also, I'd maybe avoid this particular method altogether, because you're only going to use it in the inflation procedure. It doesn't _do_ anything, it "just" fetches some values if they're there...